core-application/src/main/java/gohigher/application/port/in/DateApplicationRequest.java

```diff
@@ -0,0 +1,30 @@
+package gohigher.application.port.in;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import gohigher.application.ApplicationErrorCode;
+import gohigher.global.exception.GoHigherException;
+import lombok.Getter;
+
+@Getter
+public class DateApplicationRequest {
+	private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern(Pattern.DATE.getFormat());
+
+	private final Long userId;
+	private final LocalDate date;
+
+	public DateApplicationRequest(Long userId, String dateStr) {
```

Comment:
```suggestion
	public DateApplicationRequest(Long userId, String date) {
```
그냥 date 라고 표현해도 될 것 같습니다~!

Reply:
엇 메서드 분리로 하기전에 했던 것이....ㅎ반영하겠습니다!